src/pages/CatIndex.js

```diff
@@ -1,47 +1,67 @@
 import React from "react"
-import lawyer from "../images/lawyer-small.jpg"
-import greenEyes from "../images/green-eyes-small.jpg"
+import { NavLink } from "react-router-dom"
+import { Card, CardBody, CardTitle, CardSubtitle, Button, Row, Col } from "reactstrap"
+
+
+const CatIndex = ({ cats }) => {
 
-const CatIndex = () => {
     return (
-        <div class="container">
-            <div class="row"style={{justifyContent: "center", paddingTop: "50px", paddingBottom: "50px"}}>
-                <div class="col-sm-3">
-              
-                    <div class="card">
-                        <img class="card-img-top" src={lawyer} alt="Card image cap"  />
-                        <div class="card-body">
-                            <h5 class="card-title">Card title</h5>
-                            <p class="card-text">This is a wider card with supporting text below as a natural lead-in to additional content. This card has even longer content than the first to show that equal height action.</p>
-                            <p class="card-text"><small class="text-muted">Last updated 3 mins ago</small></p>
-                        </div>
-                    </div>
-                    </div>
-                    <div class="col-sm-3">
-                    <div class="card">
-                        <img class="card-img-top" src={greenEyes} alt="Card image cap" />
-                        <div class="card-body">
-                            <h5 class="card-title">Card title</h5>
-                            <p class="card-text">This is a wider card with supporting text below as a natural lead-in to additional content. This card has even longer content than the first to show that equal height action.</p>
-                            <p class="card-text"><small class="text-muted">Last updated 3 mins ago</small></p>
-                        </div>
-                    </div>
-                    </div>
-                    <div class="col-sm-3">
-                    <div class="card">
-                        <img class="card-img-top" src={greenEyes} alt="Card image cap" />
-                        <div class="card-body">
-                            <h5 class="card-title">Card title</h5>
-                            <p class="card-text">This is a wider card with supporting text below as a natural lead-in to additional content. This card has even longer content than the first to show that equal height action.</p>
-                            <p class="card-text"><small class="text-muted">Last updated 3 mins ago</small></p>
-                        </div>
-                    </div>
-                    </div>
-                
-            </div>
-        </div>
+    <>
+     <h1 style={{
+                        textAlign: "center", paddingTop: "20px"
+                        }}
+                        >Meet the Cats</h1>
```

Comment:
Please fix indentation for all nested data. This minor fix adds great quality to the code. Here are some different examples are indentation. The most important factor is to ensure that if the opening and closing tags are on different lines that they line up.
```javascript
<h1 
    style={{textAlign: "center", paddingTop: "20px"}}
>
    Meet the Cats
</h1>
```
```javascript
<h1 
    style={{
        textAlign: "center", 
        paddingTop: "20px"
    }}
>
    Meet the Cats
</h1>
```